roachtest: improve obs in acceptance/gossip/restart

tbg opened the pull request:
In a test, we see occasional delays to propagate updated settings within 10s. This is right following a sequence of node restarts, so it makes sense that rangefeeds would be a little slow. This logging can potentially help.

Closes https://github.com/cockroachdb/cockroach/issues/151022.

Epic: none

tbg committed:
settingswatcher: add a verbose log

tbg committed:
roachtest: add verbose logging in acceptance/gossip/restart

Touches #151022.

tbg committed:
rangefeedcache: add a verbose log line

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/151192)
<!-- Reviewable:end -->

tbg committed:
roachtest: bump to vmodule=*=1 for gossip/restart

The test does not load the cluster, so might as well err on
the side of logging too much. This is likely going to help
with the next round of investigations.

arulajmani reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> Reviewed 1 of 1 files at r1, 1 of 1 files at r2, 1 of 1 files at r3, 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/151192#-:-OXPT_B79uobnfjXvfkl:bnfp4nl)* status: <span>:shipit:</span> complete! 1 of 0 LGTMs obtained (waiting on @stevendanna)



<!-- Sent from Reviewable.io -->

arulajmani commented:
bors r+
